docs/Makefile

```diff
@@ -17,4 +17,5 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
+	python3 format_howtos.py
```

Comment:
I don't think travis uses the make file currently it probably should know that it actually does something more than calling sphinx

Reply:
OK I'll look at the travis script as well.